### DISCUSSION TITLE
SharePoint Online Site Rename

### DISCUSSION THREAD

**Discussion by YuriySamorodov** [pnp/powershell#601] [Q&A] [answered]
Hi,

Is there anyway to rename a SharePoint Online site via PnP.PowerShell the same way SharePoint Online Admin Center allows to?
https://global-sharepoint.com/sharepoint-online/how-to-change-site-url-in-sharepoint-online/

![image](https://user-images.githubusercontent.com/5260172/115420696-073ded80-a204-11eb-8af8-71293540893d.png)

Thanks in advance,
Yuriy

**Comment by LeonArmston** [comment, marked as answer]
Hi Yuri 

I dont think there is a way with PnP - you would have to use the SPO PowerShell module to do it via PowerShell (Start-SPOSiteRename) or as youu mentioned the UI in the admin centre.

**Reply by YuriySamorodov** [reply]
Thanks @LeonArmston
